language-detection-fasttext/src/main/java/com/hpe/caf/languagedetection/fasttext/LangEncoding.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright 2015-2019 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.languagedetection.fasttext;
+
+public enum LangEncoding
+{
+    // all ISO codes (184) not just ones supported by fastText (176)
```

Comment:
This list does not contain some of the languages mentioned in the fasttext documentation.
for example arz, ast, bar
https://fasttext.cc/blog/2017/10/02/blog-post.html#iso-codes-of-languages-supported

Worth looking at com.neovisionaries.i18n.LanguageCode which is used in Grammar Service, consider removing this enum and using theirs.

Reply:
```com.neovisionaries.i18n.LanguageAlpha3Code``` works like a charm here, thanks. Removed this enum class.